docs/ref/checks.txt

```diff
@@ -664,4 +664,4 @@ The following checks are performed on your URL configuration:
   including a ``:``. Remove the colon, to avoid ambiguous namespace
```

Comment:
W002 is also triggered for paths. Also, all of the above point to `django.conf.urls` instead of `django.urls`.